### PR TITLE
Add statsd instrumentation

### PR DESCRIPTION
**Pull request by SonicWang**

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -9,6 +9,10 @@ log.level = Logger::INFO unless opts['debug']
 
 opts['log'] = log
 
+# prepare statsd
+require 'statsd-ruby'
+$STATSD = Statsd.new(:host => opts['statsd_host'], :port => opts['statsd_port'])
```
Comment (jtai) [inline config.ru]: Are these going to error out if these options aren't passed?

**Comment by jtai** [comment]
:speedboat:

File: config.ru (hunk 1 above)
Comment (schleyfox) [inline config.ru]: This should just be a constant like `STATSD`, global variables should only be used when the value will vary, and probably shouldn't be used.

**Comment by schleyfox** [comment]
I'm not 100% sure that the STATSD client is thread safe, but it's probably close enough.  :+1:

**Pull request merged**